extensions/common.go

```diff
@@ -76,7 +76,8 @@ func statsReporterHandleNotificationSuccess(statsReporters []interfaces.StatsRep
 	}
 }
 
-func statsReporterHandleNotificationFailure(statsReporters []interfaces.StatsReporter, game string, platform string, err *errors.PushError) {
+func statsReporterHandleNotificationFailure(
```

Comment:
Was this change done because of line length? If so, it is better to format the function like this:
```
fund abc(
    param1 string,
    param2 int,
) (int, error) {
    // code....
}
```

Reply:
I will modify it accordingly.